packages/assistify-ai/server/lib/SmartiAdapter.js

```diff
@@ -34,22 +34,93 @@ export class SmartiAdapter {
 	 */
 	static onMessage(message) {
 
-		//TODO is this always a new one, what about update
-		const helpRequest = RocketChat.models.HelpRequests.findOneByRoomId(message.rid);
-		const supportArea = helpRequest ? helpRequest.supportArea : undefined;
-		const requestBody = {
-			// TODO: Should this really be in the responsibility of the Adapter?
-			webhook_url: SmartiAdapter.rocketWebhookUrl,
-			message_id: message._id,
-			channel_id: message.rid,
-			user_id: message.u._id,
-			// username: message.u.username,
-			text: message.msg,
-			timestamp: message.ts,
-			origin: message.origin,
-			support_area: supportArea
+		//TODO trigger on message update, if needed
+		const requestBodyMessage = {
```

Comment:
I believe we should also propagate the type (`t`) in order to properly react on messages created by the system